database/package.json

```diff
@@ -23,6 +23,7 @@
     "fix": "npm-run-all -l -s lint-fix format-fix"
   },
   "dependencies": {
+    "@swc/core": "^1.3.76",
```

Comment:
:heart_eyes_cat:

Reply:
I did have to take it out of the API unfortunately, for now at least. As it was causing our unit tests to break. But working fine in the app (and database, tho it probably doesn't matter)